Ginger/Ginger/AutomatePageLib/AddActionMenu/ApplicationModels/POMNavPage.xaml.cs

```diff
@@ -140,99 +119,25 @@ private void MainTreeView_ItemSelected(object sender, SelectionTreeEventArgs e)
                 xPOMSplitter.IsEnabled = false;
             }
-        }
+        }        
 
         public void RefreshTreeItems(object sender, RoutedEventArgs e)
         {
             UpdatePOMTree();
         }
 
-        private void SetElementsGridView()
-        {
-            xMainElementsGrid.SetTitleLightStyle = true;
-            GridViewDef view = new GridViewDef(GridViewDef.DefaultViewName);
-            view.GridColsView = new ObservableList<GridColView>();
-
-            view.GridColsView.Add(new GridColView() { Field = nameof(ElementInfo.ElementTypeImage), Header = " ", StyleType = GridColView.eGridColStyleType.ImageMaker, WidthWeight = 5, MaxWidth = 16 });
-            view.GridColsView.Add(new GridColView() { Field = nameof(ElementInfo.ElementName), Header = "Name", WidthWeight = 25, AllowSorting = true });
-
-            List<GingerCore.GeneralLib.ComboEnumItem> ElementTypeList = GingerCore.General.GetEnumValuesForCombo(typeof(eElementType));
-            view.GridColsView.Add(new GridColView() { Field = nameof(ElementInfo.ElementTypeEnum), Header = "Type", WidthWeight = 15, AllowSorting = true, StyleType = GridColView.eGridColStyleType.ComboBox, CellValuesList = ElementTypeList });
-
-            view.GridColsView.Add(new GridColView() { Field = "", Header = "Highlight", WidthWeight = 10, AllowSorting = true, StyleType = GridColView.eGridColStyleType.Template, CellTemplate = (DataTemplate)this.PageGrid.Resources["xHighlightButtonTemplate"] });
-            //view.GridColsView.Add(new GridColView() { Field = nameof(ElementInfo.IsAutoLearned), Header = "Auto Learned", WidthWeight = 10, MaxWidth = 100, AllowSorting = true, ReadOnly = true });
-            xMainElementsGrid.SetAllColumnsDefaultView(view);
-            xMainElementsGrid.InitViewItems();
-            xMainElementsGrid.ChangeGridView(eGridView.RegularView.ToString());
-
-            xMainElementsGrid.AddToolbarTool(eImageType.GoBack, "Add to Actions", new RoutedEventHandler(AddFromPOMNavPage));
-        }
-
-        private void AddFromPOMNavPage(object sender, RoutedEventArgs e)
-        {
-            if (xMainElementsGrid.Grid.SelectedItems != null && xMainElementsGrid.Grid.SelectedItems.Count > 0)
-            {
-                foreach (ElementInfo elemInfo in xMainElementsGrid.Grid.SelectedItems)
-                {
-                    ActionsFactory.AddActionsHandler(elemInfo, mContext);
-                }
-            }
-            else
-                Reporter.ToUser(eUserMsgKey.NoItemWasSelected);
-        }
-
-        private void HighlightElementClicked(object sender, RoutedEventArgs e)
-        {
-            if (!ValidateDriverAvalability())
-            {
-                return;
-            }
-
-            if (mSelectedElement != null)
-            {
-                mWinExplorer.HighLightElement(mSelectedElement, true);
-            }
-        }
-        private bool ValidateDriverAvalability()
```

Comment:
Why it shows functions like ValidateDriverAvalability as deleted?

Reply:
ValidateDriverAvalability() has moved to the ActionsPOMListViewHelper.cs